### PR TITLE
Closes #114

### LINKED ISSUE #114: Create Dataset Loader for ChEBI (NACTEM)
Task: NER, RE
License: Creative Commons 4.0
Format: brat
Language: English
Citation: @InProceedings{Shardlow2018,
  author     = {Shardlow, M J and Nguyen, N and Owen, G and O'Donovan, C and Leach, A and McNaught, J and Turner, S and Ananiadou, S},
  booktitle  = {Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)},
  title      = {A New Corpus to Support Text Mining for the Curation of Metabolites in the {ChEBI} Database},
  year       = {2018},
  month      = may,
  pages      = {280--285},
  conference = {Eleventh International Conference on Language Resources and Evaluation (LREC 2018)},
  language   = {en},
  location   = {Miyazaki, Japan},
}

Source: http://www.nactem.ac.uk/chebi/ChEBI.zip

### PR DESCRIPTION
**Pull request by mart1nro**
Note: Two offsets for an entity are defined as discontinuous spans in the dataset. This causes text_by_offset errors when running the tests.
e.g.
```
Example:174 - entity:87_T51  text:`saturated fatty acyl chains` != text_by_offset:`saturated`
Example:174 - entity:87_T51  text:`` != text_by_offset:`fatty acyl chains`
```

### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.

### CODE REVIEW

File: biodatasets/chebi_nactem/chebi_nactem.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+            doc_file = os.path.join(data_dir, doc_id + ".txt")
+            with open(doc_file) as handle:
+                text = handle.read()
+
+            ann_file = os.path.join(data_dir, doc_id + ".ann")
+
+            annotations = []
+            relations = []
+
+            with open(ann_file) as handle:
```
Comment (sg-wbi) [inline biodatasets/chebi_nactem/chebi_nactem.py:181]: Could you please switch to the functions for parsing BRAT files found in  `utils.parsing`? It's just to be 100% sure.

**Review by sg-wbi — COMMENTED** [review]
@mart1nro Thank you for your contribution? Could you please check my comment?

File: biodatasets/chebi_nactem/chebi_nactem.py (hunk 1 above)
Comment (mart1nro) [reply biodatasets/chebi_nactem/chebi_nactem.py]: Done.

**Comment by sg-wbi** [comment]
Just as a future reference, many entities seem to have an extra whitespace:
```
Example:12 - entity:6_T35  text:`masoprocol` != text_by_offset:`masoprocol  `
Example:12 - entity:6_T15  text:`glucose` != text_by_offset:`glucose  `
```

**Comment by sg-wbi** [comment]
Great job @mart1nro! Let's close this one! Thank you for your contribution!

**Pull request merged**